bin/zField

```diff
@@ -150,8 +156,37 @@ if __name__ == '__main__':
             dMapDict = clusters.makeDensityMap(RADeg, decDeg, galaxyCatalog, z, dz, rMaxMpc = 1.5)
             astImages.saveFITS(args.outDir+os.path.sep+"densityMap_z%s_%.3f_%.3f.fits" % (z, RADeg, decDeg),
                                dMapDict['map'], dMapDict['wcs'])
-    
+            
+    # Estimate errors for density map, CS and A
+    if ErrMap == True:
+        print("Starting MC 1 of %..."%(MC))
+        AErr = [] ; CSErr = [] ; MapErr=[]
+        newGalaxyCatalog=galaxyCatalog
+        for monteCarlo in range(0,MC):
+            if monteCarlo%100==0:
+                print("************** cycle %s ****************"%(monteCarlo))
+            for g in newGalaxyCatalog:
+                for i in bands:
+                    res = g.get(i, None)
+                    if res and i[-3:]!="Err":
+                        g[i]=g[i]+ np.random.normal(0,g[i+'Err']) # Add Gaussian noise to photmetry of each galaxy
+            
+            if fitZPOffsets == True:
+                photoRedshiftEngine.calcZeroPointOffsets(newGalaxyCatalog)
+            photoRedshiftEngine.calcPhotoRedshifts(newGalaxyCatalog, calcMLRedshiftAndOdds = True) # Recompute photo-z's
+            
+            dMapDictwerr = clusters.makeDensityMap(RADeg, decDeg, newGalaxyCatalog, z, dz, rMaxMpc = 1.5) # Make new density maps with added noise
+            CSErr.append(dMapDictwerr['CS'])
+            AErr.append(dMapDictwerr['A'])
+            MapErr.append(dMapDictwerr['map'])
+        AError=np.percentile(AErr, 68.3) # 68th percentile as error
+        CSError=np.percentile(CSErr, 68.3)
+        MapError=np.percentile(MapErr,68.3, axis=0)
+        astImages.saveFITS(args.outDir+os.path.sep+"errorMap_z%s_%.3f_%.3f.fits" % (z, RADeg, decDeg),
+                               MapError, dMapDict['wcs'])
+        
     # We may as well print some stats
     print(">>> Summary:")
     print("    N = %d" %  (len(tab['zPhot'])))
     print(">>> Total time: {:.1g} minutes".format((time.time()-start)/60.))
+    print("CS:\t"+str(dMapDict['CS'])+" +- "+str(CSError)+"\nA:\t"+str(dMapDict['A'])+" +- "+str(AError))
```

Comment:
You assume here that `CSError` and `AError` are defined. But they are not if [args.ErrMap == False](url). So you need to handle that case. Otherwise, this throws an exception.